css/selectors/nth-child.json

```diff
@@ -88,8 +88,8 @@
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": false,
-                "notes": "See <a href='https://bugzil.la/854148'>bug 854148</a>."
+                "version_added": "112",
+                "notes": "Available only in Nightly builds."
```

Comment:
`of <selector>` syntax is available in the beta channel as of `113.0b2` ([intent to ship](https://groups.google.com/a/mozilla.org/g/dev-platform/c/eQeeFKYJ038), enabled by default in [bug 1808229](https://bugzilla.mozilla.org/show_bug.cgi?id=1808229)).

Reply:
Thanks for letting me know